### PR TITLE
[test-framework] Various cleanup for port forwarder

### PR DESCRIPTION
**Pull request by nmittler**

### CODE REVIEW

File: pkg/test/kube/rest.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+
+	kubelib "istio.io/istio/pkg/kube"
+)
+
+// newRestClient creates a rest client with default config.
+func newRestClient(kubeconfig, configContext string) (*rest.RESTClient, *rest.Config, error) {
```
Comment (ozevren) [inline pkg/test/kube/rest.go]: Can you put these in config.go?

Comment (nmittler) [reply pkg/test/kube/rest.go]: done.

**Pull request merged**